package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "neomad.org",
+  "description": "Neomad is a open-source community project for nomad workers and people willing to _nomadize_ themselves.",
+  "main": "index.js",
+  "scripts": {
+    "build": "webpack"
```

Comment:
I'm not sure that just `webpack` would be the production version for running the build.
I see `webpack -p` [here](https://webpack.js.org/guides/production-build/#the-automatic-way).
Another documentation: http://moduscreate.com/optimizing-react-es6-webpack-production-build/
We [don't want source map](http://www.pro-react.com/materials/appendixA/#generatingsource) in production.

What is the final filesize of your js build (_/static/components/App.js_)?

This is the webpack world that I – for the same of my brain sanity – remain away from… 👻